flutter/lib/src/integrations/flutter_error_integration.dart

```diff
@@ -60,11 +60,6 @@ class FlutterErrorIntegration extends Integration<SentryFlutterOptions> {
         final mechanism = Mechanism(
           type: 'FlutterError',
           handled: true,
-          data: {
```

Comment:
Not really needed anymore, people are used to scrolling down and reading all the available context.